specification/migrate/resource-manager/readme.md

```diff
@@ -20,6 +20,21 @@ batch:
   - tag: package-migrateengine-2022-05
 ```
 
+### Tag: package-preview-2023-09
+These settings apply only when `--tag=package-preview-2023-09` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2023-09'
+input-file:
+  - Microsoft.Migrate/AssessmentProjects/preview/2023-09-09-preview/migrate.json
+suppressions:
+  - code: AvoidAdditionalProperties
```

Comment:
>AvoidAdditionalProperties [](http://example.com/codeflow?start=10&length=25)

ARM is trying to reduce instances of unschematized APIs to make it easier for consumers of those APIs to understand them and use them reliably, not to mention provide better support in the various SDKs.

Please add for this suppresions the from and where properties to be easier to review them. - [https://github.com/Azure/autorest/blob/main/docs/generate/suppress-warnings.md](https://github.com/Azure/autorest/blob/main/docs/generate/suppress-warnings.md)

Reply:
"additionalProperties" is already being used in multiple places in the existing version. This is not added in any new property.
Example: under "AKSAssessmentSummary".
Also, please check this currently open swagger PR for 2023-05-01-preview api version which is using these same suppressions:
https://github.com/Azure/azure-rest-api-specs/pull/28274/files#diff-082d114eea0a501f98601de0e2ad319db2a01dc0fc39cdbbb905bd40fbbcbc69

Reply:
why it was added if the current pr is not adding a new one?

Reply:
This is already being added in the other PR I mentioned. 
I added in this PR also so that lint errors can be resolved. Once that PR is merged, Will be merging those changes and remove from the current PR.